src/mail.py

```diff
@@ -0,0 +1,87 @@
+import email
+import imaplib
+import os
+import smtplib
+import ssl
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from os.path import basename
+from typing import List
+
+COMPANY_MAIL = os.environ.get("COMPANY_MAIL", "")
+PASSWORD = os.environ.get("PASSWORD", "")
+
+if COMPANY_MAIL == "":
+    print("Environmental variable SERVER_GMAIL not set. Sending mail won't function!")
+if PASSWORD == "":
+    print("Environmental variable SERVER_GMAIL_PASSWORD not set. Sending mail won't function!")
+
+
+def SendMail(receiver: str, subject: str, content: str, files: List[str]) -> None:
+    """
+    function which sends mail to appropriate address
```

Comment:
1. Function names in Python should generally be in snake case (SendMail -> send_mail)
2. I would remove the "function which", "function used for", etc. from all docstrings, because it's obvious that it is a function